### ISSUE TITLE
Two Modules Within Same Extension

### ISSUE THREAD

**Issue by dlight20** [elegantthemes/create-divi-extension#36]
### Problem Description
When I add a second module under the [extension name]/includes/modules, the second module loaded from within VB doesn't display fields defined in the get_fields function.  The second module is present in VB, just the properties in get_fields are not displayed.

I've played around with this a little bit and noticed it isn't class specific, rather only the first class loaded from the loader gets the additional needed custom fields.  The second classes fields never get displayed from within the builder.

I'm not sure where to go from here, wondering if anyone else has experienced this?

### Steps To Reproduce
1.

**Comment by xxtesaxx** [comment]
Had a similar issue when adding new fields to my modules. Usually a local `window.localStorage.clear()` on the browsers console together with a force reload (holding shift while reloading) does the trick.

**Comment by VladET** [comment]
@dlight20 Could you share a .zip with your extension?

**Comment by dlight20** [comment]
@VladET 

[test-multi-modules.zip](https://github.com/elegantthemes/create-divi-extension/files/1955186/test-multi-modules.zip)

This is just an example - if you try to view fields for the second module they will not be displayed.  To get the file uploaded, I removed the node_modules libraries so you'll need to re-import them.  

I appreciate any guidance you can provide.

**Comment by VladET** [comment]
@dlight20 Thank you, that is perfect. It's working great on my end. 

![multiple-modules](https://user-images.githubusercontent.com/1061516/39364926-547204aa-4a38-11e8-9834-23de6e345078.gif)

Do you see any errors in your terminal?

**Comment by dlight20** [comment]
@VladET 

From the VB screen above mine is working as expected too.  I have attached an image displaying where I am seeing the issue, I apologize for a possible terminology mistake, I am a newbie to WordPress/Divi...

![screen shot 2018-04-27 at 9 50 49 am](https://user-images.githubusercontent.com/9876300/39366028-c7ee6c3a-4a00-11e8-8a71-76b3e747c9b5.png)


Thanks again!

**Comment by xxtesaxx** [comment]
Have you tried to clear your local storage?

![image](https://user-images.githubusercontent.com/3000808/39366109-471bb3dc-4a33-11e8-9f6f-747213c57d3a.png)

**Comment by randomblink** [comment]
So for clarification purposes, if I intended to create multiple modules that were all complimentary of each other, I could just put them all in an 'extension' plugin? Which technically means if I intended to develop Custom Post Types with Custom Divi Modules I could just include them ALL in one extension because Extensions are also Plugins?

**Comment by xxtesaxx** [comment]
Yes, you can use the plugin file in the root folder for your functions (e. g. adding post types, meta boxes etc.) and the /includes/modules/ can be filled with as many Divi modules as you desire
